recipes/_config.rb

```diff
@@ -13,3 +13,10 @@
     node.override['hops']['capacity']['resource_calculator_class'] = "org.apache.hadoop.yarn.util.resource.DominantResourceCalculatorGPU"    
   end
 end
+
+# Override install binaries for enterprise edition
+if node['install']['enterprise']['install'].casecmp? "true"
+  node.override['hops']['dist_url']    = "#{node['install']['enterprise']['download_url']}/hopshadoop/hops-#{node['hops']['version']}.tgz"
```

Comment:
If there is no reason, can you remove the `hopshadoop` directory to avoid confusion? It should be the mirror of CE

Reply:
limitation of archive.logicalclocks.com. Can not copy files to root dir